atomic_reactor/plugins/post_koji_import.py

```diff
@@ -83,7 +82,6 @@
 ]
 
 
-@label('koji-build-id')
```

Comment:
We need this one (could be in annotations instead, maybe)
https://github.com/containerbuildsystem/atomic-reactor/blob/c0659607cebd90ead55e53591b6c027e3b6fdd34/tekton/tasks/source-container-set-results.yaml#L32
https://github.com/containerbuildsystem/atomic-reactor/blob/c0659607cebd90ead55e53591b6c027e3b6fdd34/tekton/tasks/binary-container-set-results.yaml#L34

Reply:
so change it to annotations and change it also in those tasks as that is required